src/components/LandingHeroBackgroundMap/index.tsx

```diff
@@ -75,7 +43,8 @@ export const LandingHeroBackgroundMap: FC<LandingHeroBackgroundMapPropType> = ({
         clickHandler={() => undefined}
         markers={[
           {
-            ...locationViewport,
+            latitude: sensor.latitude || 0,
```

Comment:
It feels wrong to have a fallback value for lat/lon.

@ff6347, perhaps we could already make this non-nullable in the DB?

Reply:
I agree, I had to do it because the types indicated them as optional. I would vote for making them mandatory in the DB too.

Reply:
> perhaps we could already make this non-nullable in the DB?

Should we? What if the user does not know the location yet? What if he does not want to publish the location (maybe its in his home)? What if I want to prepare some sensors and position them later on? 

Also setting the lat/lon to 0/0 puts the sensor somewhere west of Afrika. So we shouldn't do that either. Keep them (null in the DB and undefined in the frontend and don't display them if lat or lon is missing.

Reply:
> Should we?

My impression so far was that a sensor without a geolocation is contrary to the idea of Stadtpuls. But let's not discuss this deep down in a PR review and instead talk about it next week in our checkin